pages/account/index.tsx

```diff
@@ -94,6 +94,59 @@ export default function MyAccount({
       } else {
         alert(`error: ${responseData}`)
       }
+    } catch (error) {
+      console.log(error)
+      alert("fetching error")
+    }
+  }
+
+  /** Testing hold update api route */
+  async function holdUpdate(patronId, holdId, freeze, pickupLocation) {
```

Comment:
since holdId, freeze, and pickupLocation are all being passed to sierra, can they be combined into one object here? My thinking was to have this method pass along the update request body without having to know what's necessarily in it.

Reply:
oh i see this is just the test button!